tournament-init.yaml

```diff
@@ -0,0 +1,12 @@
+!tournament
+id_ctr: 6
+players:
+  0: !player {aga_id: 12345, name: Andrew, rank: -1, mm_score: 0}
+  1: !player {aga_id: 12390, name: Walther, rank: 5, mm_score: 0}
+  2: !player {aga_id: 234, name: Milan, rank: -6, mm_score: 0}
+  3: !player {aga_id: 2399, name: James, rank: -10, mm_score: 0}
+  4: !player {aga_id: 2, name: Quinten, rank: 3, mm_score: 0}
+  5: !player {aga_id: 4, name: Neil, rank: 2, mm_score: 0}
+rounds: []
+old_pairs: !!set {}
+current_players: !!set {}
```

Comment:
Can also `git checkout -- tournament.yaml` to reset it to what it's committed as.